Examples/UIExplorer/StatusBarExample.js

```diff
@@ -293,6 +305,21 @@ exports.examples = [{
   },
 }];
 
+if (android) {
```

Comment:
Can't we just use the `platform` parameter that we also use in ImageExample? That would fix the `ios &&` , `android &&` code above as well.